k8s_helm_charts/zdm/templates/configmap.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ include "zdm.fullname" . }}
+  namespace: {{ .Values.namespace }}
+data:
+  ZDM_ORIGIN_PORT: "9042"
+  ZDM_TARGET_PORT: "9042"
+  ZDM_PRIMARY_CLUSTER: ORIGIN
+  ZDM_READ_MODE: PRIMARY_ONLY
+  ZDM_LOG_LEVEL: INFO
+  ZDM_PROXY_MAX_CLIENT_CONNECTIONS: "1000"
+  ZDM_METRICS_ENABLED: "true"
```

Comment:
should some of this be helm values? I thought ZDM had a couple different modes and you wouldn't it clobbered on an upgrade?

Reply:
Yeah that's a good reminder. I definitely should address this.

Reply:
Should be addressed in the latest commit.